### PR TITLE
CDPT-1909 - fixes styled dropdowns.

### PR DESCRIPTION
**Pull request by EarthlingDavey**
- Pulls in latest ACF from the official repo.
- Adds ACF key during build and runtime.
- Fixes styled dropdowns.

### CODE REVIEW

File: bin/composer-auth.sh [hunk 1]
```diff
@@ -20,8 +20,9 @@ then
 	cat <<- EOF >> auth.json
 		{
 			"http-basic": {
-			  "composer.wp.dsd.io": {"username": "$COMPOSER_USER","password": "$COMPOSER_PASS"},
-        "composer.deliciousbrains.com": {"username": "$AS3CF_PRO_USER","password": "$AS3CF_PRO_PASS"}
+				"composer.wp.dsd.io": {"username": "$COMPOSER_USER","password": "$COMPOSER_PASS"},
+				"composer.deliciousbrains.com": {"username": "$AS3CF_PRO_USER","password": "$AS3CF_PRO_PASS"},
+				"connect.advancedcustomfields.com": {"username": "$ACF_PRO_LICENSE", "password": "https://intranet.justice.gov.uk"}
```
Comment (wilson1000) [inline bin/composer-auth.sh:25]: Hi @EarthlingDavey - Did you set up basic authentication using our domain as a password?

**Review by wilson1000 — APPROVED** [review]
Just one comment my friend. Thanks for this.

File: bin/composer-auth.sh (hunk 1 above)
Comment (wilson1000) [reply bin/composer-auth.sh:25]: Our security department will flag this as the 'password' has a value. We might want to stop them from messaging us by popping the value in a hidden variable.

Comment (EarthlingDavey) [reply bin/composer-auth.sh:25]: Hey, good spot. I didn't set the password, it's how acf does the auth.

https://www.advancedcustomfields.com/resources/installing-acf-pro-with-composer/#build-systems-and-environment-variables

Do you think commenting would be sufficient, or set an environment variable?

**Pull request merged**